### PR TITLE
fix import log dirctory to: github.com/casbin/log

### PR DESCRIPTION
**Pull request by CHCP**
I think the old code is error: ``github.com/casbin/casbin/log``,   I fix to ``github.com/casbin/log``.

### CODE REVIEW

File: enforcer.go [hunk 1]
```diff
@@ -20,13 +20,13 @@ import (
...
 	"github.com/Knetic/govaluate"
 	"github.com/casbin/casbin/effect"
-	"github.com/casbin/casbin/log"
 	"github.com/casbin/casbin/model"
 	"github.com/casbin/casbin/persist"
 	"github.com/casbin/casbin/persist/file-adapter"
 	"github.com/casbin/casbin/rbac"
 	"github.com/casbin/casbin/rbac/default-role-manager"
 	"github.com/casbin/casbin/util"
+	"github.com/casbin/log"
```
Comment (hsluoyz) [inline enforcer.go]: What is ``github.com/casbin/log``?

**Comment by hsluoyz** [comment]
You can still send PR for the other commit.

**Comment by CHCP** [comment]
Yes, I will send PR for ``GetUsersForRoleInDomain``, but the test is failed. I think the function: ``GetUsersForRoleI`` does not work. I don't know why, I am not familiar with casbin core, would you like finding this reason?

**Comment by hsluoyz** [comment]
@CHCP , can you first rework your commits in this PR? The commits in a PR can be modified. You can remove your 1st and 3rd commit about ``github.com/casbin/log``. Then I will see why it fails based on the latest PR.

**Comment by CHCP** [comment]
I reset the git commits, and all commits remove(1st, 2nd, 3rd). So I open a new PR.

**Pull request closed without merging**